jitbuilder/release/.gitignore

```diff
@@ -0,0 +1,33 @@
+###############################################################################
+#
+# (c) Copyright IBM Corp. 2016, 2016
+#
+#  This program and the accompanying materials are made available
+#  under the terms of the Eclipse Public License v1.0 and
+#  Apache License v2.0 which accompanies this distribution.
+#
+#      The Eclipse Public License is available at
+#      http://www.eclipse.org/legal/epl-v10.html
+#
+#      The Apache License v2.0 is available at
+#      http://www.opensource.org/licenses/apache2.0.php
+#
+# Contributors:
+#    Multiple authors (IBM Corp.) - initial implementation and documentation
+###############################################################################
+
+include/compiler/
```

Comment:
I should have broken my questions up more clearly.  

Ok thanks, I missed that the omr/jitbuilder/release folder was currently present in the source tree.

I agree that having a whitelist is not the right answer.  My second question was really attempting to ask if we should just put these entries in the top level .gitignore?  This is more of a general question about how the project wants to handle .gitignore files.  Do we only want one top level file or do we want component level files?